src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -163,89 +155,125 @@ SelectClause Visitor::visitTypesafe(Parser::SelectClauseContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-VarOrAlias Visitor::visitTypesafe(Parser::VarOrAliasContext* ctx) {
+VarOrAlias Visitor::visit(Parser::VarOrAliasContext* ctx) {
   return visitAlternative<VarOrAlias>(ctx->var(), ctx->alias());
 }
 
 // ____________________________________________________________________________________
-Alias Visitor::visitTypesafe(Parser::AliasContext* ctx) {
+Alias Visitor::visit(Parser::AliasContext* ctx) {
   // A SPARQL alias has only one child, namely the contents within
   // parentheses.
-  return visitTypesafe(ctx->aliasWithoutBrackets());
+  return visit(ctx->aliasWithoutBrackets());
 }
 
 // ____________________________________________________________________________________
-Alias Visitor::visitTypesafe(Parser::AliasWithoutBracketsContext* ctx) {
-  return {{visitTypesafe(ctx->expression())}, visitTypesafe(ctx->var())};
+Alias Visitor::visit(Parser::AliasWithoutBracketsContext* ctx) {
+  return {{visit(ctx->expression())}, visit(ctx->var())};
 }
 
 // ____________________________________________________________________________________
-ParsedQuery Visitor::visitTypesafe(Parser::ConstructQueryContext* ctx) {
+ParsedQuery Visitor::visit(Parser::ConstructQueryContext* ctx) {
   if (!ctx->datasetClause().empty()) {
     reportError(ctx->datasetClause(0),
                 "QLever currently doesn't support FROM clauses");
```

Comment:
Ok, can you go over the error messages and use a consistent pattern? I would prefer
`<FEATURE> (is|are) currently not supported by QLever`. Then the more important part stands at the beginning and this would be:
`FROM clauses are currently not supported by QLever`.